docker-compose.dev.yml

```diff
@@ -11,24 +11,24 @@
 version: '3.4'
 
 services:
-  reaction-admin:
+  admin:
     # The main `docker-compose.yml` has an `image` prop. Unfortunately, when we
     # add `build` prop here, it changes the meaning of that `image` prop to
     # "tag the built image with this image name". This has the effect of breaking
     # the app after you've run with the override and then go back to running without
-    # it, because now `reactioncommerce/admin:trunk` would actually be your dev image.
+    # it, because now `admin:trunk` would actually be your dev image.
     # To work around this issue, we specify a different tag name here, which does not
     # match any of our published tags.
-    image: reactioncommerce/admin:local-dev
+    image: mailchimp-open-commerce/admin:local-dev
     build:
       context: .
       dockerfile: Dockerfile-dev
-    command: bash -c "export PATH=$PATH:/home/node/.meteor && npm install --no-audit && node ./.reaction/waitForMongo.js && node --experimental-modules ./.reaction/scripts/run.mjs"
+    command: bash -c "export PATH=$PATH:/home/node/.meteor && npm install --no-audit && node ./.moc/waitForMongo.js && node --experimental-modules ./.moc/scripts/run.mjs"
```

Comment:
I think you didn't rename the `.reaction` folder to `.moc`.
Make sure to think of the effects this would cause as this is the folder where the settings are stored. So it will have to be updated at other places also.

Reply:
True - do you think we should we take the opportunity to rename Reaction related folders in this round or should we save for future iterations?

Reply:
Let's just rename `.reaction` for now.

Reply:
Cool - so we'll save the internal naming for the future then.